src/Microsoft.Win32.Registry/src/Configurations.props

```diff
@@ -2,8 +2,8 @@
 <Project ToolsVersion="14.0" DefaultTargets="Build" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
   <PropertyGroup>
     <BuildConfigurations>
-      netcoreapp-Windows_NT;
-      netcoreapp-Unix;
+      netcoreapp2.0-Windows_NT;
+      netcoreapp2.0-Unix;
       netstandard;
```

Comment:
It's actually not "most".  I preferred to target `netcoreapp2.0` specifically rather than cross-compile for most packages.  I did this for the same reason many of the packages still maintain live builds of old netstandard TFMs: these things aren't part of the platform.  They are packages on top of the platform and shouldn't cross-compile just because the platform versions.  I don't want to set precedence that causes the package to bloat with every framework version.

I made a couple exceptions where we already had implementations making use of new 'netcoreapp2.1' surface area.